srcs/database/Dockerfile.dev

```diff
@@ -1,23 +1,24 @@
-# syntax=docker/dockerfile:1
 FROM    alpine:3.16
 
-# args 
-ARG     POSTGRES_USER=${POSTGRES_USER}
-ARG     POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
-ARG     POSTGRES_DB=${POSTGRES_DB}
-ARG     PGDATA=${PGDATA}
-
-# install MariaDB from package manager and shadow (login and pass utilities)
-RUN     apk update && apk upgrade && apk add --update --no-cache \
-            postgresql14 \
-            icu-data-full \
- &&     rm -f /var/cache/apk/* \
-\
-# configuration
- &&     mkdir -p "$PGDATA" \
- &&     chmod 777 "$PGDATA"
-
-# launch command as root
-EXPOSE  5432
-
-CMD     ["postgres"]
\ No newline at end of file
+COPY ./conf/dev-mock-data.sql ./
+
+COPY ./scripts/docker-entrypoint.dev.sh ./
+
+ENV POSTGRES_USER=transcendence
+ENV POSTGRES_PASSWORD=5euApbMoLVtNgEcKs44e
+ENV POSTGRES_DB=transcendence_db
```

Comment:
We should not have these variables directly written in the docker file. To be updated.